stylesheets/components/cards.css

```diff
@@ -0,0 +1,172 @@
+.card-vertical{
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    flex-basis: 13rem;
+    padding: 1rem;
+    border-radius: 0.375rem;
+    position: relative;
+}
+
+.card:hover{
+    box-shadow: 0 2px 8px;
+}
+
+.card-badge-green, .card-badge-purple, .card-badge-grey{
+    padding: 0.3rem 0.5rem;
+    color: white;
+    position: absolute;
+    top: 1rem;
+}
+
+.card-badge-green{
+  background-color: green;
+}
+
+.card-badge-purple{
+  background-color: purple;
+}
+
+.card-badge-grey{
+  background-color: grey;
+}
+
+.card-img{
+    height: 100%;
+    width: 100%;
+    object-fit: fit;
+    mix-blend-mode: multiply; /* merges background color of image with div background */
+}
+
+.card-img-box{
+    background-color: white;
+    height: 14rem ;
+    width: 12rem;
+}
+
+.card-title{
+    display: flex;
+    align-items: center;
+    margin: 0.5rem 0;
+    gap: 0.6rem;
+}
+.card-title h4{
+    font-size: 1rem;
+    font-weight: var(--wgt-bold);
+}
+
+.card-ratings{
+    display: flex;
+    align-items: center;
+    background-color: green;
+    padding: 0.5rem 0.7rem ;
+    margin-bottom: 0.5rem;
+    color: white;
+    font-size: 1rem;
+    gap: 0.3rem;
+    font-weight: var(--wgt-bold);
+}
+
+.card-ratings .material-icons{
+    font-size: 1rem;
+}
+
+.card-pricing{
+  display: flex;
+  gap: 0.5rem;
+  font-weight: var(--wgt-bold);
+  font-size: 1.2rem;
+  
+}
+
+.card-price-cut{
+    color: grey;
+    text-decoration: line-through;
+}
+
+.card-percent-discount{
+    color: green;
+}
+
+.card .btn{
+    margin: 0.5rem 0;
+    width: 12rem;
+    justify-content: center;
+}
+
+.card-dismiss-btn{
+    position: absolute;
+    top: 0.5rem;
+    right: 0.5rem;
+    background-color: lightgray;
+    border-radius: 50%;
+    padding: 6px;
+    cursor: pointer;
+}
+
+.card-overlay-container{
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(0,0,0,0.5);
+    border-radius: 0.375rem;
+    z-index: 2;
+    cursor: context-menu;
+}
+
+.card-overlay-text{
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    background-color: #dee2e6;
```

Comment:
Try to use variables for colors as much as possible.

Reply:
Thanks Siddhant!! made suggested changes on line 125. Please approve my PR.